.github/workflows/autorelease.sh

```diff
@@ -0,0 +1,68 @@
+if [ $# -lt 1 ]; then
+    echo -e "Missing parameter. Usage: ./autorelease [github_token]"
+    exit 1
+fi
+
+CHANGELOG_FILE="CHANGELOG.md"
+
+git fetch --prune --unshallow --tags
+
+# Sort tags by date and pick the last one
+LAST_TAG_RAW=$(git for-each-ref --sort=creatordate --format '%(refname:short)' refs/tags | tail -1)
```

Comment:
You should be able to get the tag as an environment variable in the github job and pass it to this function